dashboard/menu/dashboard/pretranslate.md

```diff
@@ -1,14 +1,14 @@
 # Pre-translate
 
-By using the Pre-translate menu, you can initiate a process which supplies untranslated segments with a preliminary translation in order to mitigate bleedthrough on the proxied pages. You can use pre-translate with a Translation Memory or a Machine Translation API of your choice. the translation proxy supports Google Translate, iTranslate4eu, Bing Translate and GeoFluent.
+By using the Pre-translate menu, you can initiate a process which supplies untranslated segments with a preliminary translation in order to mitigate bleedthrough on the proxied pages. You can use pre-translate with a Translation Memory as well as any of the supported Machine Translation APIs.
 
-In order to use Machine Translation, you need to go into your Account and set up your MT account Ids/authorization keys in the "Machine Translation" menu.
+In order to use Machine Translation, you need at least one API configured for your project. You can access configuration for available Machine Translators by clicking (configure) next to the desired Trasnlator, or by selecting them on in the menu bar.
```

Comment:
Consider using italics to distinguish control labels - simple parens de-emphasize content.

Reply:
On the ui it's lin parenthesen, i used it as a visual guide. Understand your point, i switched to italic